src/set.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+export default class Set {
+  constructor(initialValues=[]) {
+    this.elements = initialValues
+  }
+
+  add(value) {
+    if (this.elements.indexOf(value) === -1) {
+      this.elements[this.elements.length] = value
+      return this.elements
+    }
+    return false
+  }
+
+  isEmpty() {
+    if (this.elements.length === 0 ){
+      return true
+    }
+    return false
+
+
+  }
+
+  contains(value) {
+    if (this.elements.indexOf(value) > 0) {
+      return true
+    }
+    return false
+  }
+
+  remove(value) {
+    if (this.elements.indexOf(value) > 0) {
+      return this.elements.splice(this.elements.indexOf(value),1)
+    }
+    return "The value to remove does not exist"
+  }
+
+  forEach(func) {
+    this.elements = this.elements.map(func)
+    return this.elements
+  }
```

Comment:
I feel like this should be implemented differently for learning purposes.  You have essentially used Map to implement Map.